src/java.base/windows/classes/sun/nio/fs/WindowsFileSystem.java

```diff
@@ -42,12 +43,12 @@
     private final String defaultRoot;
 
     // package-private
-    WindowsFileSystem(WindowsFileSystemProvider provider,
-                      String dir)
+    WindowsFileSystem(WindowsFileSystemProvider provider)
     {
```

Comment:
The "{" can be moved to the previous line now.

Reply:
Cleaned up in 66788f5d74ff9088b08159ff3ccd4b49cdaf6bbb.